docs/applications/link-server-to-application.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Link a Server to an Application
+intro: Link a server to an application to select which servers an application can be deployed.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/configuration-files/add-configuration-file
+    next:
+        - /docs/applications/deploy-application
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to link a server
+1. On the upper-right corner of the application details view, click `...`
+1. Use the drop-down menu to choose `Linked resources`
+1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
+1. Follow the prompts then click `Link`
+1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table
```

Comment:
`find the selected server`? Why? For what? 

If the intention is to have a closing statement, it should not be a "step/order" to the reader, should just be an informational note
```
> In the <element>, the <info> can be seen in <info location>
> In the <element>, the <info> will be displayed in <info location>
etc
```

If it's an imperative instruction, it needs to guide the user o do something with the info that was found
```
find <info> to <...>
```

Reply:
Post updated